src/langchain_google_alloydb_pg/engine.py

```diff
@@ -586,11 +608,16 @@ async def _aload_table_schema(
         metadata = MetaData()
         async with self._engine.connect() as conn:
             try:
-                await conn.run_sync(metadata.reflect, only=[table_name])
+                await conn.run_sync(
+                    metadata.reflect, schema=schema_name, only=[table_name]
+                )
             except InvalidRequestError as e:
-                raise ValueError(f"Table, {table_name}, does not exist: " + str(e))
+                raise ValueError(
+                    f"Table, {table_name}, does not exist in schema {schema_name}: "
```

Comment:
nit: f"Table, '{table_name}', does not exist in schema, '{schema_name}': " or 
f"Table,'{schema_name}'. '{table_name}', does not exist : "